From 140476c3e6a6f497bdd95c0c039cb9a680f5a414 Mon Sep 17 00:00:00 2001
From: Andre McCurdy <armccurdy@gmail.com>
Date: Fri, 22 Jan 2016 15:53:04 -0800
Subject: [PATCH] parallel build fix

When statically linking the sqlite3 binary, ensure that it depends on
its own version of sqlite3.o, instead of the version being built (in
parallel) for use in the libsqlite3.a static library.

Upstream-Status: Pending

Signed-off-by: Andre McCurdy <armccurdy@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index e9b3950..0e8f043 100644
--- a/configure.ac
+++ b/configure.ac
@@ -129,7 +129,7 @@ AC_ARG_ENABLE(static-shell, [AS_HELP_STRING(
   [statically link libsqlite3 into shell tool [default=yes]])], 
   [], [enable_static_shell=yes])
 if test x"$enable_static_shell" == "xyes"; then
-  EXTRA_SHELL_OBJ=sqlite3.$OBJEXT
+  EXTRA_SHELL_OBJ=sqlite3-sqlite3.$OBJEXT
 else
   EXTRA_SHELL_OBJ=libsqlite3.la
 fi
-- 
1.9.1

